.github/actions/setup-and-build/action.yml

```diff
@@ -7,6 +7,13 @@ runs:
       shell: bash
       run: |
         ./scripts/install.sh
+        source ./scripts/devenv.sh
+        # NOTE(vadorovsky): For some reason, Github runners sometimes don't give
+        # a heck about rust-toolchain.toml files and keep using random versions
+        # of rustc.
+        rustup update
```

Comment:
Why do we need this command? I'm wondering too. I have absolutely no idea. But without it, CI still keeps using some outdated Rust 1.67.0. Even though I'm doing the override and rust-toolchain.toml. WTF

I wish I had infinite amount of time on debugging that properly, but seems like it's going to be a less pending TODO.

Reply:
could this have to do with buildjet instead of github?